src/components/Modal/index.js

```diff
@@ -188,6 +200,8 @@ Modal.propTypes = {
      * @ignore
      */
     children: PropTypes.node,
+    /** Determines if the cross for close is showed or not */
+    showClose: PropTypes.bool,
```

Comment:
the most use cases will use the close button so instead of use showClose I think it is better to use hideCloseButton to be more explicit, also this way is more consistent with what we do in Notification component. Then default value would be:
```hideCloseButton: false,```
So let's change the description to:
```If true, hide the close button in the modal```
as we did in Notification

Reply:
I have added this suggestion, thank you very much!